React/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -1,30 +1,49 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import { formValueSelector } from 'redux-form';
 
 import routes from '../../../constants/routes';
-import actions from '../../../redux/login/actions';
+import LoginActions from '../../../redux/login/actions';
 
 import LoginForm from './layout';
+import { NAME_EMAIL, NAME_PASS } from './components/fieldNames';
 
 class Login extends Component {
   handleSubmit = () => {
-    this.props.setLoggedUser(true);
+    this.props.login({ email: this.props.email, password: this.props.pass });
     this.props.history.push(routes.APP);
   };
 
   render() {
-    return <LoginForm onSubmit={this.handleSubmit} />;
+    return (
+      <Fragment>
+        <div className="auth-error">{this.props.authError}</div>
+        <LoginForm onSubmit={this.handleSubmit} />
+      </Fragment>
+    );
   }
 }
 
-const mapDispatchToProps = {
-  setLoggedUser: actions.setLoggedUser
-};
+const selector = formValueSelector('loginForm');
```

Comment:
this is not wrong.. but there is another way to take the inputs value in the form. I do want you to check it out because it's much simplier. Just try to receive params in the handleSubmit method and print them to see what is inside. There is not need to change this in the PR! just see what happend

Reply:
oh! it's really simple!! things that I love of high level programming languages :heart: